packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -311,12 +317,96 @@ try {
 				// @TODO: Do not hardcode this
 				processApi.stdout(`18 140`);
 				processApi.exit(0);
-			} else if (command[0] === 'less') {
+			} else if (args[0] === 'less') {
 				processApi.on('stdin', (data: Uint8Array) => {
 					processApi.stdout(data);
 				});
 				processApi.flushStdin();
 				processApi.exit(0);
+			} else if (args[0] === 'php') {
+				if (!childPHP) {
+					childPHP = new WebPHP(await recreateRuntime(), {
+						documentRoot: docroot,
+						absoluteUrl: scopedSiteUrl,
+					});
+					// Pretend to use the CLI SAPI to bypass the check done by WP-CLI.
+					childPHP.setSapiName('cli');
+					// Share the parent's MEMFS instance with the child process.
+					// Only mount the document root and the /tmp directory,
+					// the rest of the filesystem (like the devices) should be
+					// private to each PHP instance.
+					for (const path of [docroot, '/tmp']) {
+						if (!childPHP.fileExists(path)) {
+							childPHP.mkdir(path);
+						}
+						childPHP[__private__dont__use].FS.mount(
+							childPHP[__private__dont__use].PROXYFS,
+							{
+								root: path,
+								fs: php[__private__dont__use].FS,
+							},
+							path
+						);
+					}
+				}
+
+				let result: PHPResponse | undefined = undefined;
+				try {
+					// @TODO: Run the actual PHP CLI SAPI instead of
+					//        interpreting the arguments and emulating
+					//        the CLI constants and globals.
+					const cliBootstrapScript = `<?php					
+					// Set the argv global.
+					$GLOBALS['argv'] = array_merge([
+						"/wordpress/wp-cli.phar",
+						"--path=/wordpress"
+					], ${phpVar(args.slice(1))});
+			
+					// Provide stdin, stdout, stderr streams outside of
+					// the CLI SAPI.
+					define('STDIN', fopen('php://stdin', 'rb'));
+					define('STDOUT', fopen('php://stdout', 'wb'));
+					define('STDERR', fopen('/tmp/stderr', 'wb'));
+
```

Comment:
Should this be `/internal/stderr`?

Reply:
Oh! Yeah if that remains a file path then yes. I wonder why this isn't `php://stderr` actually, I think I copied it over from https://github.com/WordPress/wordpress-playground/pull/161 and, at that time, there was a bug preventing the use of `php://stderr`.

Reply:
I created an issue to track it here: https://github.com/WordPress/wordpress-playground/issues/1071